ruff_lsp/server.py

```diff
@@ -421,6 +423,47 @@ def code_action(params: CodeActionParams) -> list[CodeAction] | None:
                         ),
                     )
 
+    # Add "Disable for this line" for every diagnostic.
```

Comment:
We probably need to return the `noqa_line` for each diagnostic, since this won't work correctly for multi-line strings.

Reply:
Thanks for the pointer, I'll try adding it.